micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,98 @@
+        }
+        return messageMap;
+      },
+    },
+  };
+
+  const { data: localizationMap, isLoading: isLocalizationLoading } = Digit.Hooks.useCustomAPIHook(localizationReqCriteria);
+
+  // Transform data if localizationMap is available
+  let finalData = data;
+  if (data && data.data && localizationMap && localisableFields?.length > 0) {
+    let updatedData = _.cloneDeep(data);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'updatedData' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)